Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2.yml

```
@@ -78,21 +88,21 @@ script:
       name: limit
     - auto: PREDEFINED
       defaultValue: last_seen
-      description: Sorting based on the column(last_seen,first_seen,ioc_type)
+      description: Sorting based on the column(last_seen,first_seen,ioc_type).
       name: sort_by
       predefined:
       - last_seen
       - first_seen
       - ioc_type
     - auto: PREDEFINED
       default: true
-      defaultValue: desc
-      description: A sorting order for ioc. Either Ascending or Descending. The default value is Descending.
+      defaultValue: asc
+      description: A sorting order for ioc. Either Ascending or Descending. The default value is Ascending.
```

```suggestion
      description: A sorting order for ioc. 
```
I removed the options and the default value from the description since when you generate the readme from the yml file, these values are automatically added.  If we have it in the description in the yml file, then the information will be doubled in the readme.